src/fheroes2/dialog/dialog_selectscenario.cpp

```diff
@@ -631,43 +658,41 @@ const Maps::FileInfo * Dialog::SelectScenario( const MapsFileInfoList & allMaps,
         else if ( le.isMouseRightButtonPressedInArea( buttonSelectAll.area() ) ) {
             ShowToolTip( _( "All Maps" ), _( "View all maps, regardless of size." ) );
         }
-        else if ( le.isMouseRightButtonPressedInArea( countPlayers ) || le.isMouseRightButtonPressedInArea( curCountPlayer ) ) {
-            ShowToolTip( _( "Players Icon" ),
-                         _( "Indicates how many players total are in the scenario. Any positions not occupied by human players will be occupied by computer players." ) );
+        else if ( le.isMouseRightButtonPressedInArea( countPlayers ) ) {
+            ShowIfFound( listbox, le.getMouseCursorPos(), []( const Maps::FileInfo * info ) { PlayersToolTip( info ); } );
```

Comment:
There (and in the rest of the similar cases below) is no need in lambda. Just pass the (pointer to) function directly:

```suggestion
            ShowIfFound( listbox, le.getMouseCursorPos(), PlayersToolTip );
```

Reply:
Done